# Request fields
int8       interaction_id
---
# Response fields
float32[]  covariance
float32[]  mean
